assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.test.js

```diff
@@ -285,7 +285,8 @@ describe( 'WidgetAreaRenderer', () => {
 
 		await waitFor( () => {
 			expect( widgets ).toHaveLength( 1 );
-			expect( container.querySelectorAll( '.googlesitekit-widget-area' ) ).toHaveLength( 0 );
+			expect( container.querySelectorAll( '.googlesitekit-widget-area' ) ).toHaveLength( 1 );
+			expect( container.querySelector( '.googlesitekit-widget-area' ) ).toHaveClass( 'googlesitekit-hidden' );
```

Comment:
I think this change matches the new spec!  (From looking at the comments on the ticket)

Reply:
Yes, correct 👍